configs/sandbox_demo/bad_actor.conf

```diff
@@ -9,4 +9,7 @@ setenv FOO "foo"
 source configs/sandbox_demo/security_workshop_demo_common.conf
 
 new-window -n bad_actor
-send-keys 'ros2 run example_nodes cpu_hog 8'
+send-keys 'ros2 run example_nodes cpu_hog 32'
+
+split-window -v
+send-keys 'ros2 launch example_nodes sandboxed_cpu_hog.launch.py'
```

Comment:
We could spawn a new window and run top / glances automatically.

Reply:
That'd be interesting...

Reply:
This is an extension of the main configuration, so the glances window titled "diagnostics" is running.

Reply:
Do we need this in the readme then?

>If you run the top command (`ros2 run...`), you can see the CPU of your system jump to 100% usage, grinding everything else to a halt.

Just wondering if the info is already present in glances.

Reply:
Yeah the info is already there - and it's called out higher up in README in the intro to the overall demo. I've just pushed a commit with a little hint to remember that it exists.